lib/pages/forms_pages/components/biting_location_form.dart

```diff
@@ -47,7 +47,8 @@ class _BitingLocationFormState extends State<BitingLocationForm> {
                   Utils.report!.selected_location_lon!)));
           currentLocation = Position(
               latitude: Utils.report!.selected_location_lat!,
-              longitude: Utils.report!.selected_location_lon!);
+              longitude: Utils.report!.selected_location_lon!,
+              accuracy: -1, altitude: -1, altitudeAccuracy: -1, speed: -1, speedAccuracy: -1, heading: -1, headingAccuracy: -1, timestamp: null);
```

Comment:
I got the error that all parameters are required and I could not compile otherwise. I am not sure why this was not happening before, probably some version/dependencies stuff
```
The named parameter 'speed' is required, but there's no corresponding argument.
Try adding the required argument.dart[missing_required_argument](https://dart.dev/diagnostics/missing_required_argument)
```
![image](https://github.com/Mosquito-Alert/Mosquito-Alert-Mobile-App/assets/157690872/355b5b3e-4f6d-4dc9-8e74-b52b90224a58)

Reply:
Ah, that's interesting. It appears that the `Position` class from the `geolocator` package wasn't designed to be instantiated directly. It serves as the type of object returned by methods like `getCurrentPosition` or `getLastKnownPosition`. Considering this, it might be more appropriate to replace them all and utilize the `LatLng` class from `google_maps_flutter.dart` instead, which seems to be used elsewhere in the project. However, making this change is significant and requires thorough testing on the device to ensure that everything continues to function as intended.

Reply:
yup, LatLng seems more appropriate when the other parameters are not used

> However, making this change is significant and requires thorough testing on the device to ensure that everything continues to function as intended.

Are we doing it in a future version then?

Reply:
Could you try replacing all "Position" instances for LatLng? Maybe it's just that simple. Test it in your emulator just to be sure, but we can test it later once the draft is released

Reply:
I just tried it and seems to be working on my emulator. You can see it in the latest commit "Replace Position for LatLng"